domains/playboyfear.json

```
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Fear-Itself",
+           "email": "",
+           "discord": "1079471323220295781",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.lxlc-07Kcz9oxe3endfLP2iaYnhFx1YdAtlEl8n0GHWodaqDj6rscZ-tyqOgcQLgH4juWf4tm8f3ExHzYrwx097K5X97Rf3ZvibOI5KlhKwNxMDEPSrsaq13r2ILCAZGmeQmVnNVWzmbiwDns-z_uh0QEl-YvGatR2Z5XhNfvID_dRtshwkqdwi9SsDuwX6Y2qHUBBTtuy0a35nBU9iITFMLHKSvkmfnul6hwRo6g9rJhUZIebynuUCwDxb9seba8kI1tWYBGXNE0uwGb-1SJKp4XrFTsLZYlMXBCTdwCFtebONQtWAkbkuQAfhUuqtOvIbz6KcIyI7g6KV5OnPNzw.q9Fb1LgcuGOhDM7d8UPiuw.fdmJ9W-LPDrT6p53ITGhZqNxbisbRcssIA9XUoFvbrDCk8apYtDAeIaqR3N5QWtydIcKVkPdvhbEU-Dduga5PXqsO9Ot1MQLeErtI28O1H8.pKzhEjWsJBuESoPUMQzM4g"
+        },
+    
+        "record": {
+            "CNAME": "fear-itself.github.io/"
```

CNAME cannot contain `/`.

```suggestion
            "CNAME": "fear-itself.github.io"
```